alacritty_terminal/src/ansi.rs

```diff
@@ -983,6 +989,30 @@ where
                 unhandled(params);
             },
 
+            // Hyperlink.
+            b"8" => {
+                if params.len() >= 3 {
+                    let (link_params, uri) = (params[1], params[2]);
+
+                    if uri.is_empty() {
+                        self.handler.end_hyperlink();
+                        return;
+                    }
+
+                    // Link parameters are in format of `key1=value1:key2=value2`.
+                    // Currently only key `id` is defined.
+                    let id = link_params
+                        .split(|&b| b == b':')
+                        .find(|kv| kv.starts_with(b"id="))
+                        .and_then(|kv| str::from_utf8(&kv[3..]).ok());
+                    // Is empty URI a sane default?
```

Comment:
Not starting a hyperlink if we don't understand the URI is probably a saner default.